Shared/Samples/Identify features in WMS layer/IdentifyFeaturesInWMSLayerView.swift

```diff
@@ -0,0 +1,112 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct IdentifyFeaturesInWMSLayerView: View {
+    /// A map with a dark gray base basemap centered on the USA.
+    @State private var map: Map = {
+        let map = Map(basemapStyle: .arcGISDarkGrayBase)
+        let center = Point(x: -95.7129, y: 37.0902, spatialReference: .wgs84)
+        map.initialViewpoint = Viewpoint(center: center, scale: 7e7)
+        return map
+    }()
+    
+    /// The WMS layer with EPA water info.
+    @State private var waterInfoLayer: WMSLayer?
+    
+    /// The text of a WMS feature HTML attribute that is shown in the web view.
+    @State private var webViewText = String()
```

Comment:
```suggestion
    @State private var webViewText = ""
```

I think we generally try to initialize strings like this.